src/features/device-state/index.ts

```diff
@@ -23,9 +23,11 @@ export {
 	serviceInstallFromImage,
 } from './state-get-utils';
 export {
-	metricsPatchFields,
+	validDeviceMetricsRecordPatchFields,
```

Comment:
NIT: Since this field is also exported from index.ts w/ `export * as deviceState from './features/device-state';`, how about not renaming it so that we don't introduce a breaking change?